sdk/src/sysvar/clock.rs

```diff
@@ -1,28 +1,28 @@
-//! This account contains the current slot, epoch, and stakers_epoch
+//! This account contains the clock slot, epoch, and stakers_epoch
 //!
 use crate::account::Account;
-use crate::syscall;
+use crate::sysvar;
 use bincode::serialized_size;
 
 pub use crate::timing::{Epoch, Slot};
 
-crate::solana_name_id!(ID, "Sysca11Current11111111111111111111111111111");
+crate::solana_name_id!(ID, "SysvarC1ock11111111111111111111111111111111");
```

Comment:
tiny nit, could move this down below the def of ID so its consistent with the other syscall files.